src/jit/lower.cpp

```diff
@@ -265,7 +265,7 @@ void Lowering::DecomposeNode(GenTreePtr* pTree, Compiler::fgWalkData* data)
         {
             GenTree* nextTree = tree->gtNext;
             GenTree* rhs = tree->gtGetOp1();
-            if (rhs->OperGet() == GT_PHI)
+            if (rhs->OperGet() == GT_PHI || rhs->OperGet() == GT_CALL)
```

Comment:
A comment here would also be useful, to explain why the `GT_CALL` case requires no decomposition.